.github/workflows/ci-actions.yml

```diff
@@ -7,6 +7,7 @@ on:
     paths-ignore:
       - 'docs/src/main/asciidoc/**'
       - 'README.md'
+      - 'COMMITTERS.adoc'
```

Comment:
Great. I suppose we should ignore it https://github.com/quarkusio/quarkus/blob/95afcf6fc9d9422c34be9e2f108849b801a01211/.github/workflows/ci-actions.yml#L24 too?

Reply:
@gsmet WDYT?

Reply:
Ah right, that's why CI got triggered! Fixed.